pallets/asset-manager/src/lib.rs

```diff
@@ -551,6 +565,25 @@ pub mod pallet {
             });
             Ok(())
         }
+
+        /// Set min xcm fee for asset/s on their reserve chain.
+        ///
+        /// * `origin`: Caller of this extrinsic, the access control is specified by `ForceOrigin`.
+        /// * `filtered_location`: Multilocation to be filtered.
+        #[pallet::call_index(6)]
+        #[pallet::weight(T::WeightInfo::update_outgoing_filtered_assets())]
+        #[transactional]
+        pub fn update_outgoing_filtered_assets(
+            origin: OriginFor<T>,
+            filtered_location: T::Location,
+        ) -> DispatchResult {
+            T::ModifierOrigin::ensure_origin(origin)?;
+            FilteredOutgoingAssetLocations::<T>::insert(filtered_location.clone().into(), ());
```

Comment:
Also there isn't really a way to remove an item from this piece of storage. 

Maybe we should add a way to also remove item from `FilteredOutgoingAssetLocations`.

Reply:
Yeah of-course there should be a way ... will add.

Reply:
How about this ?